quest/mp2.py

```diff
@@ -10,12 +10,24 @@ def df_mp2(wfn):
     pass
 
 
-def _mo_transform():
+def _mo_transform(orbital_energies):
+    
     pass
 
 
-def _denom():
-    pass
+def _denom(fock_matrix):
+    #get energies from fock matrix)
+    #multiply C and F to get 
+    eps = wfn.arrays.get("EPSILON", None )
+    if eps != None:
+        nocc = 5 #must pull from wavefunction when implemented
```

Comment:
Lets try and avoid inline comments, if you really want them we should keep them pep8 certified. so they should be formatted `#  comment text` not `#comment text`.